packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx

```diff
@@ -15,12 +15,13 @@ import { HorizontalRadioGroup, TheatreAudioPlayer, TheatreTrackList } from '../.
  * @param { string } channel
  * @param { string } labelValue
  */
-const getChannelLabelToDisplay = ({ channel, labelValue }) => {
+const getChannelLabelToDisplay = ({ channel, labelValue, title }) => {
   const label = <span className="channel-label">{ labelValue }</span>;
+  const titleToDisplay = title || `playing ${labelValue}`;
```

Comment:
QUESTION: Do we need this extra default value? We pass in a correct value of e.g. "play YouTube" to `getChannelLabelToDisplay()`. Then we have this default of e.g. "playing YouTube", which is different. And *then* we have a default value in the icon component itself e.g. "YouTube logo". Seems like one too many defaults, esp. since this default above is *inconsistent* from our desired e.g. "play YouTube". I'd probably get rid of this default above and just use `title`.

Reply:
oh yes thanks for the catch. will remove